src/poliastro/threebody/restricted.py

```diff
@@ -0,0 +1,173 @@
+# coding: utf-8
+
+import numpy as np
+from scipy.optimize import root
+
+
+def lagrange_points(r12, m1, m2):
+    """Computes the Lagrangian points using the formulation
```

Comment:
Could we have a short line first, then more explanation in the body? As in https://numpydoc.readthedocs.io/en/latest/format.html#sections